hibernate-cache/src/main/java/org/hibernate/cache/infinispan/util/LockOwner.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.cache.infinispan.util;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+
+import org.infinispan.commands.CommandInvocationId;
+
+final class LockOwner {
+
+	private LockOwner() {
+	}
+
+	static void writeTo(ObjectOutput out, Object lockOwner) throws IOException {
```

Comment:
That comes from the command's key lock owner, which normally is the command invocation id.